src/resources/js/pages/system/localisation/EditTexts.vue

```diff
@@ -198,10 +198,12 @@ export default {
                 return this.sortedKeys;
             }
 
-            const self = this;
+            const query = this.query.toLowerCase();
 
-            return this.langKeys.filter(key => key.toLowerCase()
-                .indexOf(self.query.toLowerCase()) > -1);
+            return this.langKeys.filter(key => {
+                return key.toLowerCase().indexOf(query) > -1
+                || (this.langFile[key] && this.langFile[key].toLowerCase().indexOf(query) > -1);
+            });
```

Comment:
I'm not sure what you mean with selecting `&&`, `this.langFile[key] && ` is there in case certain keys are missing; otherwise the entire page would fail (due to `.toLowerCase()` not being applicable to an undefined object)

Reply:
Resolved 
(just FYI: I give maintainers access to my patches so that things like this can be modified directly)